README.md

```diff
@@ -79,7 +79,38 @@ from .env file, Otherwise app will not work locally.
 
 ## Installing the bot on a repository
 
-You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/settings/apps/my-oppiabot-testing-app/installations`. In the `.env` file put your github account name in `WHITELISTED_ACCOUNTS` and also add your repository (name in small caps) in the `constants.js` file locally.
+
+1. Identify a target repository and install the bot by clicking the Install button on the settings page of your app, e.g., **`https://github.com/settings/apps/my-oppiabot-testing-app/installations`** 
+2. Update the **`.env` file**:
+   - Add your GitHub account name to `WHITELISTED_ACCOUNTS`.  
+3. Add the repository name in the **`constants.js` file**:  
```

Comment:
Please add a blank line above "3." and above "2.", similar to how you have a space between sections 3. and 4.

Reply:
![Screenshot 2024-12-10 185802](https://github.com/user-attachments/assets/b76baca9-4734-4057-9863-38f9527154ef)

@seanlip will this work?